[2022-07-28 13:57:59,491] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v5.upload_to_s3_2019-01 manual__2022-07-28T13:57:40.027582+00:00 [queued]>
[2022-07-28 13:57:59,545] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: dataquality_pipeline_v5.upload_to_s3_2019-01 manual__2022-07-28T13:57:40.027582+00:00 [queued]>
[2022-07-28 13:57:59,545] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 13:57:59,545] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-07-28 13:57:59,545] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-07-28 13:57:59,625] {taskinstance.py:1397} INFO - Executing <Task(LocalFilesystemToS3Operator): upload_to_s3_2019-01> on 2022-07-28 13:57:40.027582+00:00
[2022-07-28 13:57:59,648] {standard_task_runner.py:52} INFO - Started process 11247 to run task
[2022-07-28 13:57:59,671] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dataquality_pipeline_v5', 'upload_to_s3_2019-01', 'manual__2022-07-28T13:57:40.027582+00:00', '--job-id', '220', '--raw', '--subdir', 'DAGS_FOLDER/dataquality_pipeline_v5.py', '--cfg-path', '/tmp/tmp7s4h3s7v', '--error-file', '/tmp/tmphkjatwlh']
[2022-07-28 13:57:59,674] {standard_task_runner.py:80} INFO - Job 220: Subtask upload_to_s3_2019-01
[2022-07-28 13:57:59,974] {task_command.py:371} INFO - Running <TaskInstance: dataquality_pipeline_v5.upload_to_s3_2019-01 manual__2022-07-28T13:57:40.027582+00:00 [running]> on host 2c2de32fb47f
[2022-07-28 13:58:00,364] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=abin404@yopmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dataquality_pipeline_v5
AIRFLOW_CTX_TASK_ID=upload_to_s3_2019-01
AIRFLOW_CTX_EXECUTION_DATE=2022-07-28T13:57:40.027582+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-07-28T13:57:40.027582+00:00
[2022-07-28 13:58:00,405] {base.py:68} INFO - Using connection ID 'aws-conn' for task execution.
[2022-07-28 13:58:00,407] {base_aws.py:206} INFO - Credentials retrieved from login
[2022-07-28 13:58:02,006] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=dataquality_pipeline_v5, task_id=upload_to_s3_2019-01, execution_date=20220728T135740, start_date=20220728T135759, end_date=20220728T135802
[2022-07-28 13:58:02,139] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-07-28 13:58:02,311] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
